src/utilities/getDeposit/getDeposit.ts

```diff
@@ -17,61 +25,59 @@ async function getDefaultDeposit() {
 }
 
 async function getDepositWeb3Name(
-  did: DidUri | undefined,
+  did: Did | undefined,
 ): Promise<DepositData | undefined> {
   if (!did || !isFullDid(did)) {
     return getDefaultDeposit();
   }
 
   const api = ConfigService.get('api');
 
-  const { web3Name } = Did.linkedInfoFromChain(
-    await api.call.did.query(Did.toChain(did)),
-  );
+  const {
+    document: { alsoKnownAs },
+  } = linkedInfoFromChain(await api.call.did.query(toChain(did)));
 
-  if (!web3Name) {
+  if (!alsoKnownAs) {
     return getDefaultDeposit();
   }
 
-  const data = await api.query.web3Names.owner(web3Name);
+  const data = await api.query.web3Names.owner(alsoKnownAs[0]);
 
   if (data.isNone) {
     return;
   }
-  return Did.depositFromChain(data.unwrap().deposit);
+  return depositFromChain(data.unwrap().deposit);
 }
 
 export function useDepositWeb3Name(
-  did: DidUri | undefined,
+  did: Did | undefined,
 ): DepositData | undefined {
   return useAsyncValue(getDepositWeb3Name, [did]);
 }
 
 export async function getDepositDid(
-  did: DidUri | undefined,
+  did: Did | undefined,
 ): Promise<DepositData | undefined> {
   const api = ConfigService.get('api');
 
-  if (!did || Did.parse(did).type === 'light') {
+  if (!did || parse(did).type === 'light') {
     if ('deposit' in api.consts.did) {
       // TODO: remove this `if` once the Spiritnet is updated
-      return { amount: api.consts.did.deposit as BN };
+      return { amount: api.consts.did.deposit as unknown as BN };
```

Comment:
It could be a good moment to do what is to be done.

Reply:
Confirm. We don't have this deposit logic anymore. It can go.